java/ql/lib/ext/java.lang.model.yml

```diff
@@ -3,6 +3,8 @@ extensions:
       pack: codeql/java-all
       extensible: sinkModel
     data:
+# suggested label is not supported:      - ["java.lang", "Module", True, "getResourceAsStream", "(String)", "", "Argument[0]", "read-file", "generated"]
```

Comment:
Is "suggested label is not supported" referring to the "read-file" kind label? This kind is supported for the path-injection query now that https://github.com/github/codeql/pull/12081 is merged, so perhaps this model can be added?

Reply:
That's exactly what it's referring to! Thank you for the correction, will fix :)